frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/GitCommandUpdatingCurrentBranchBackgroundable.java

```diff
@@ -51,7 +51,6 @@
 import com.virtuslab.qual.guieffect.UIThreadUnsafe;
 
 @ExtensionMethod(GitMacheteBundle.class)
-@CustomLog
 public abstract class GitCommandUpdatingCurrentBranchBackgroundable extends Task.Backgroundable {
```

Comment:
in particular. now the logging from derived classes will be more precise. right?

Reply:
Yes, the logger name will point to the actual runtime type, not to one of superclasses